README.md

```diff
@@ -11,10 +8,64 @@ and is based on the `(Tree)SummarizedExperiment` data container.
  
 The package can be directly installed from R command line.
 
-```
+
+```r
 devtools::install_github("microbiome/miaTime")
 ```
 
+```
+## Downloading GitHub repo microbiome/miaTime@HEAD
+```
+
+```
+## These packages have more recent versions available.
+## It is recommended to update all of them.
+## Which would you like to update?
+## 
+## 1: All                            
+## 2: CRAN packages only             
+## 3: None                           
+## 4: fansi (0.5.0  -> 1.0.2 ) [CRAN]
+## 5: Rcpp  (1.0.7  -> 1.0.8 ) [CRAN]
+## 6: rjson (0.2.20 -> 0.2.21) [CRAN]
+## 
+## fansi (0.5.0  -> 1.0.2 ) [CRAN]
+## Rcpp  (1.0.7  -> 1.0.8 ) [CRAN]
+## rjson (0.2.20 -> 0.2.21) [CRAN]
+```
```

Comment:
What is this?

Reply:
I assume it happens when I knit the file. I will keep an eye on that.